composer.json

```diff
@@ -14,6 +14,8 @@
     ],
     "require": {
         "php": "^8.1 || ^8.2",
```

Comment:
Delete it dependency. Already checked in the micro/kernel.

Reply:
@Asisyas, done.

Reply:
@Asisyas, I have added php 8.2 back as plugin supports only php 8.2 but micro/micro-kernel does support php 8.1|8.2, if anyone will use the plugin with 8.1 version, it will throw fatal, so we need to specify php version.